<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttackDialog</class>
 <widget class="QDialog" name="AttackDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttackDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titleEdit">
       <property name="placeholderText">
        <string>Title</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Base stat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="statComboBox">
       <item>
        <property name="text">
         <string>Strengh</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dexterity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Constitution</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Intelligence</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Wisdom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Charisma</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="profCheckBox">
       <property name="text">
        <string>Proficiency bonus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Bonus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="bonusSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Damage</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="damageEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Notes</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="notesEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../static/app_resources.qrc"/>
 </resources>
 <connections/>
</ui>
